node.gyp

```diff
@@ -477,20 +516,14 @@
           ],
         }],
         [ 'OS=="freebsd" or OS=="linux"', {
-          'ldflags': [ '-Wl,-z,noexecstack',
+          'ldflags': [ '-Wl,-z,noexecstack,--allow-multiple-definition',
```

Comment:
That sounds like (for example) a static library is getting linked in twice.  I'm inclined to say that `--allow-multiple-definition` is a workaround but not a proper fix.